schema-org-metadata/src/main/java/no/unit/nva/schemaorg/SchemaOrgDocument.java

```diff
@@ -68,30 +73,33 @@ private static InputStream toInputStream(PublicationResponse publication) {
     }
 
     private static String getJsonLdStringOfModel(Model result) {
-        return RDFWriter.create()
-                .format(RDFFormat.JSONLD10_FRAME_PRETTY)
-                .context(getJsonLdWriteContext(extractTypeForFrame(result)))
-                .source(result)
-                .build()
-                .asString();
-    }
-
-    private static JsonLDWriteContext getJsonLdWriteContext(String type) {
-        var context = new JsonLDWriteContext();
-        context.setOptions(getJsonLdOptions());
-        context.setFrame(createFrame(type));
-        return context;
+        var byteArrayOutputStream = new ByteArrayOutputStream();
```

Comment:
Very similar code/duplicate of code in `FramedJsonGenerator` - extract to common util?

Reply:
Different dependencies make this a job for a new PR